### ISSUE TITLE
fail to train a model

### ISSUE THREAD

**Issue by Auth-qqual** [ipc-lab/NDIC#6]
Hello,I change the dataset to train a new model.But it stuck when the epoch arrive at 1000. I mean the loss no longer decreases.So I am here to ask for your help.
![image](https://user-images.githubusercontent.com/107161727/172750155-c852d6b8-ee36-4472-84fe-882958b5bc7a.png)
![image](https://user-images.githubusercontent.com/107161727/172750045-ad28670b-b7fb-4fab-82e3-0baf0fa96cee.png)
Here are the results.
![image](https://user-images.githubusercontent.com/107161727/172750421-2f8d266d-f233-4ad6-97d7-e3cb7baf2957.png)

**Comment by garjania** [comment]
Hello. Have you tried using a different learning rate or scheduler patience/factor? Since you change the resize to 256,256, using the same configuration might not be the best option for this case. Also, there's a chance that by resizing to 256,256, the images are stretched, and this messes up the correlation between the image pairs.

**Comment by garjania** [comment]
Here are the results we obtained for a model that is trained using 256x256 images. However, in the dataloader, instead of cropping to (370, 740), the images are cropped into (370, 370) to avoid the stretching effect. Also, in this case, the learning rate is set to 0.00005
![28b5e859-814c-4c4c-a857-90982fee4148](https://user-images.githubusercontent.com/33597130/175081793-a7d00e4c-0d9f-4f30-9fc6-4b66c10376ad.jpg)
.

**Comment by ezgimez** [comment]
closing this issue. can re-open it later if there are any follow-ups.
